SDK.markdown

```diff
@@ -7,6 +7,104 @@ sidebar: true
 
 We provide you with a NPM package for Node.js and Browser environments. Please see our [GitHub page for details](https://github.com/kontist/sdk).
 
+## Getting started
+This is a very short step by step guide to help you getting started and fetch a few transactions with our API and SDK in Node.js. You can do much more with it, please refer to the documentation or [get in touch with us](mailto:developer@kontist.com).
+
+### Requirements
+Please have Node.js + NPM installed. This example is using plain JavaScript for simplicity, but you can use TypeScript as well.
+
+### Setup a new project
+Create a new folder `kontist-connect-example` and open it in your terminal.
+
+Init the project and add `kontist` as an dependency.
+```bash
+npm init -y
+npm install kontist --save
```

Comment:
Not very important, but I think we can remove the `--save` part, it's the default since npm 5: https://github.com/npm/npm/commit/c0d73dbed4f1e5699de1914c1cc291973ad87c9f